src/NServiceBus.InMemoryPersistence.AcceptanceTests/NServiceBus.InMemoryPersistence.AcceptanceTests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
If we're trying to follow downstream conventions, this project should be called:

NServiceBus.Persistence.InMemory.AcceptanceTests